Desktop version

ginabeki opened the pull request:
## Building desktop version

I did the following in this project:-
- Updated font size to match desktop version design
- Used grid layout to align items on the page
- Updated HTML markup to adjust some changes on the desktop version
- Checked for lint errors and fixed the errors
- Ensured the website is responsive

levy002 commented on styles.css:
```diff
@@ -296,7 +372,428 @@ textarea:focus {
+  }
+
+  /* Contact section style */
+  .contact {
+    margin-top: 200px;
+  }
```
- [ ] Kindly make sure to add all background images to the contact section in order to meet the provided design :wink: 

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/193150033-701bb083-802a-46d9-8c87-2f696be1eac4.png)


   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/193149980-6dab7e35-b263-4c09-8aad-1c91f767f1bb.png)

ginabeki replied:
Done

levy002 reviewed (changes requested):
## Hello @ginabeki   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ginabeki commented on index.html:
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
+    <!-- ===================Footer section===================== -->
+
+    <footer class="footer">
+      <div class="footer-wrap">
+        <p class="resume">Get my CV</p>
+        <a href="#">
+          <img class="download-icon" src="images/download.svg" alt=""
+        /></a>
+      </div>
```
Fixed

ginabeki commented on index.html:
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
           required
         ></textarea>
-        <button class="button" type="submit">Start Collaboration</button>
+        <button class="button button-contact" type="submit">
+          Start Collaboration
+        </button>
```
Fixed

ginabeki commented on index.html:
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
-      <h1>Getting Started</h1>
-      <p>
+      <h1 class="contact-header">Get Started</h1>
+      <p class="contact-text">
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
```
Doned

levy002 commented:
Thank you @ginabeki for considering all suggested changes. Kindly feel free to go ahead and submit your PR for another review. :wink:

ginabeki commented:
## Revision
Here are some of the things I added in regards to the previous feedback:-
- Added the required text color to the footer section
- Changed the width of the buttons to match the Figma design
- Changed the typography of the contact section paragraph to match the design.
- Added a missing background image to the contact section


#### Thank you for taking your time to review my work 🙌

nedjwab reviewed (approved):
# 🟢 Project Approved! 🏆

## Hi @ginabeki    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors.
- ✔good design 🔥🔥
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
